recipes/pygaps/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "pygaps" %}
+{% set version = "4.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygaps-{{ version }}.tar.gz
+  sha256: 0914aa95ebdf8aa08276f9823d42fcc02c2b48538e3dabc11d06dc25ebde08ba
+
+build:
+  skip: true  # [py<37]
```

Since this is pure Python, better to make this noarch and unconditionally depend on `importlib-resources` below:
```suggestion
  noarch: python
```